### PR TITLE
OG-720: accept non-integer in relay config (master)

### PR DESCRIPTION
**Pull request by drortirosh**

### CODE REVIEW

File: packages/dev/test/Flows.test.ts [hunk 1]
```diff
@@ -60,7 +60,7 @@ options.forEach(params => {
     before(async () => {
       await emptyBalance(gasless, accounts[0])
 
-      const gasPriceFactor = 1.2
+      const gasPriceFactor = 1
```
Comment (forshtat) [inline packages/dev/test/Flows.test.ts:63]: Is this actually related to the PR?

**Review by forshtat — APPROVED** [review]

**Review by shahafn — APPROVED** [review]
